src/stories/components/expansion-panel/ExpansionPanel.mdx

```diff
@@ -20,36 +20,45 @@ It's common to compose an expansion panel with a card component to provide a mor
 
 > **Tip:** You can provide a `<forge-open-icon>` to show the open state of the expansion panel, and the icon will be rotated when the panel is opened/closed.
 
+## Target Button
+
+A button in the header slot of the expansion panel will automatically control toggling the `open` state of the panel.
```

Comment:
May be worth adding a note about ensuring to control the existence of the trigger button and the panel in the DOM at the same time to avoid any hard to debug issues when it comes to async/incremental rendering.

These types of scenarios are why we may want to consider adding a `triggerElement` property to allow devs to set the instance directly in cases where timing can be an issue.

Eventually we can invert this process by using the Invoker Commands API with a custom command so that the button will instead notify the panel rather than the other way around which will simplify things a lot.